cost/aws/rightsize_ebs_volumes/aws_rightsize_ebs_volumes.pt

```diff
@@ -0,0 +1,1868 @@
+        label "Region"
+      end
+      field "size" do
+        label "Size (GB)"
+      end
+      field "iops" do
+        label "IOPS"
+      end
+      field "readOps" do
+        label "Read Operations"
+        path "read_ops"
+      end
+      field "writeOps" do
+        label "Write Operations"
+        path "write_ops"
+      end
+      field "utilizedOps" do
+        label "Utilized Operations"
+        path "utilized_ops"
+      end
+      field "utilizedOpsPercent" do
+        label "Utilized Operations Percent"
+        path "utilized_ops_percent"
+      end
+      field "state" do
+        label "Status"
+        path "status"
+      end
+      field "savings" do
+        label "Estimated Monthly Savings"
+      end
+      field "savingsCurrency" do
+        label "Savings Currency"
+      end
+      field "lookbackPeriod" do
+        label "Lookback Period (Days)"
+      end
+      field "service" do
+        label "Service"
+      end
+      field "id" do
+        label "ID"
+        path "volume_id"
+      end
+    end
+  end
+  validate_each $ds_underutilized_volumes_result do
+    summary_template "{{ with index data 0 }}{{ .summary_policy_name }}{{ end }}: {{ len data }} Underutilized Volumes Found"
+    detail_template <<-'EOS'
+    **Potential Monthly Savings:** {{ with index data 0 }}{{ .summary_total_savings }}{{ end }}
+
+    {{ with index data 0 }}{{ .summary_message }}{{ end }}
+    EOS
+    check logic_or($ds_parent_policy_terminated, eq(val(item, "resourceID"), ""))
+    escalate $esc_email
+    escalate $esc_delete_volumes
+    escalate $esc_modify_volumes
+    hash_exclude "age", "message", "total_savings", "tags", "savings", "savingsCurrency"
+    export do
+      resource_level true
+      field "accountID" do
+        label "Account ID"
+      end
+      field "accountName" do
+        label "Account Name"
+      end
+      field "resourceID" do
+        label "Resource ID"
+        path "volume_id"
+      end
+      field "resourceName" do
+        label "Resource Name"
+      end
+      field "tags" do
+        label "Resource Tags"
+      end
+      field "age" do
+        label "Age (Days)"
+      end
+      field "recommendationDetails" do
+        label "Recommendation"
+      end
+      field "resourceType" do
+        label "Resource Type"
+        path "volume_type"
+      end
+      field "region" do
+        label "Region"
+      end
+      field "size" do
+        label "Size (GB)"
+      end
+      field "iops" do
+        label "IOPS"
+      end
+      field "readOps" do
+        label "Read Operations"
+        path "read_ops"
+      end
+      field "writeOps" do
+        label "Write Operations"
+        path "write_ops"
+      end
+      field "utilizedOps" do
+        label "Utilized Operations"
+        path "utilized_ops"
+      end
+      field "utilizedOpsPercent" do
+        label "Utilized Operations Percent"
+        path "utilized_ops_percent"
+      end
+      field "newIops" do
```

Comment:
Might be worth asking Tatiana about this and how it impacts scraping, since the current scraper expects this to be in resourceType

Reply:
`resourceType` is mapped to volume type (above) and I think that's the strategy we would align with for all resource types where possible.

I am not certain we should use `resourceType` for new IOPS or Capacity (or any other value other than "type"/"classification"/"flavor" of the resource).... If we did use it for other values, what would we provide for `resourceType` if the resource could be rightsized in multiple ways (i.e. new iops, AND new capacity) ?

Reply:
CC @TDubovchenko -- if there's any thoughts on this consider how the UI and Recommendations scraper works

Reply:
> [Friday 1:09 PM] Tatiana Dubovchenko
> We have `newResourceType` to show in the table, it would be nice to use it somehow for EBS rightsizing. 
> Since `resourceType` will contain  volume type (gp3, io2,...) and recommendation is to keep volume type but change provisioned IOPs, should we show something like `gp3(X IOPs)` in newResourceType and also keep yours newIops column.  Then gp3(X IOPs) will be shown in the table and extract to easily eyeball it and user can see more details including new IOPs, new IOPs % in details slide out

> [Friday 1:22 PM] Bryan Karaffa
> maybe simple key=value pairs if we're going to show more than 1 new value
> `iops=3000,type=gp3`

Using this recommendation... the newResourceType will contain 1 or more `key=value` pairs (separated by comma) describing the new configuration.. It should include any unmodified configurations as well.. as an example for volumes

Reply:
I don't object to this.

Reply:
This is added.  Example here:
https://app.flexera.com/orgs/27018/automation/incidents/projects/116186?incidentId=666097225c49416bd9e45e67